docker/ubuntu-18.04.dockerfile-todo

```diff
@@ -19,8 +19,10 @@ RUN <<_DEPS
 #!/bin/bash
 apt-get update -y
 apt-get install -y --no-install-recommends \
-  software-properties-common=0.96.24.32.18
```

Comment:
Need to specify the version to keep with Docker best practices. Applies to all modified dockerfiles. Version can normally be set with an `*` but I figured this is a package that rarely, if ever changes.